lib/screens/login.dart

```diff
@@ -96,9 +102,41 @@ class _LoginState extends State<Login> {
                 ),
               ],
             ),
+            RaisedButton(
+              child: Text('TEST LCS'),
+              color: pink_dark,
+              textColor: white,
+              textTheme: ButtonTextTheme.normal,
+              elevation: 6.0,
+              shape: BeveledRectangleBorder(
+                borderRadius: BorderRadius.all(Radius.circular(3.0)),
+              ),
+              onPressed: (){this.postData();},
+            ),
           ],
         ),
       ),
     );
   }
+
+  var client = new http.Client();
+  var url = "https://7c5l6v7ip3.execute-api.us-west-2.amazonaws.com/lcs-test/authorize";
+  postData() async {
+    Map map = {
+      'email': 'f@f.com', 'password': 'f',
+    };
+    print(await apiRequest(url, map));
+  }
+
+  Future<String> apiRequest(String url, Map jsonMap) async {
```

Comment:
You can't use the LCS helper here? This looks a lot more complicated than the other one.

Reply:
Yes there's a bunch of places it could be used. I didn't want to make this merge request too big yet. At some point I'm going to wrap the lcs endpoints and we can adjust the calls then